Update login.md (#3)
cs3157-borowski/guides: login.md

# Getting Started in the UNIX Coding Environment

Welcome to COMSW 3157 Advanced Programming with Dr. Brian Borowski! Prior to this course, you may have used various coding environments like Codio, VSCode, etc. But this guide will introduce you to the one that we use for AP, the UNIX Coding Environment on our remote class server. It is very important that you read through this guide so that you will be able to complete the assignments for this course. 

You will be interacting with the UNIX coding environment via the command-line. 

## What is the "Command-Line"?

> The command-line in a UNIX coding environment allows you to interact with the operating system and execute commands. From the command line, you can navigate file systems, run programs, amongst other things. Think of this as a very simplified way of interacting with an operating system or an alternative to a mouse/trackpad. 

Reviewer: I think "basic" works better than "simplified" and I see why you changed the "or" to the "and" which sounds better but its less correct.

## Logging in to our Class Server 

> Students will recieve an account on the class server named, AP, which is a server instance running on Google Cloud Platform. Your account allows you access to a remote server where you will do all your work and save all your files. 

Reviewer: ```suggestion
Students will receive an account on the AP class server, which is an Ubuntu Linux server instance running on Google Cloud Platform. This remote server is where you will do all your work and save all your files. 
```

For those of you using macOS, we recommend using its [Terminal](https://support.apple.com/guide/terminal/welcome/mac). If you are using a Windows machine , we recommend using [cygwin](https://www.cygwin.com/), [PuTTY](https://www.putty.org/), [MobaX](https://mobaxterm.mobatek.net/), just to name a few. 

> Now, to log onto our remote class server, you will need to establish use SSH (Secure SHell).Type this into the command-line to establish your connection to our server: 

Reviewer: > you will need to establish use SSH

? Im not sure what you wanna say here

Also missing a space after the dot and before "Type"

```yaml
ssh YOUR_UNI@ap.cs.columbia.edu
```

> You should be prompted to input your password which you should have recieved in an email. Don't worry, it will seem like you are not typing anything but continue and press ENTER when done. If you followed these steps closely, you should see this message appear: 

Reviewer: ```suggestion
You should be prompted to input your password which you should have received via email. Don't worry, it will seem like you are not typing anything, but continue entering your password and press ENTER when done. If you followed these steps closely, you should see this message appear: 
```

```yaml
                  __                             __      
   _      _____  / /________  ____ ___  ___     / /_____ 
  | | /| / / _ \/ / ___/ __ \/ __ `__ \/ _ \   / __/ __ \
  | |/ |/ /  __/ / /__/ /_/ / / / / / /  __/  / /_/ /_/ /
  |__/|__/\___/_/\___/\____/_/ /_/ /_/\___/   \__/\____/ 
                                                       
                     :::     :::::::::  
                   :+: :+:   :+:    :+: 
                  +:+   +:+  +:+    +:+ 
                 +#++:++#++: +#++:++#+  
                 +#+     +#+ +#+        
                 #+#     #+# #+#        
                 ###     ### ###        

```

## Text Editors

> In this course, you will be writing code using a command line text editor on the class server. In previous semesters, most students and professors recommend Vim. But you can do your own research and use any text-editor that you feel comfortable with (for example, Emacs). 

Reviewer: I think we have to be careful here about them "doing their own research" cause we don't want them to arrive at the conclusion that they can use VSCode through SSH, so you might want to rewrite this to narrow the options.

Author: @xurxo-riesco Dr. B said students can use whatever they want. But I agree that we should refine their options to at most like 3

### Acknowledgements 
This guide was originally developed by Jae Woo Lee. 

> Adaptations by Phillip Le for Fall 23

Reviewer: ```suggestion
Adaptations by Phillip Le for Spring 23
```
